Dockerfile-2.x

```diff
@@ -0,0 +1,32 @@
+FROM centos:centos7
+MAINTAINER The ViaQ Community <community@TBA>
+
+EXPOSE 9200
+EXPOSE 9300
+USER 0
+
+ENV HOME=/opt/app-root/src \
+  JAVA_VER=1.8.0 \
+  ES_VER=2.3.3 \
+  ES_CONF=/usr/share/elasticsearch/config/elasticsearch.yml \
+  INSTANCE_RAM=8G \
+  NODE_QUORUM=1 \
+  RECOVER_AFTER_NODES=1 \
+  RECOVER_EXPECTED_NODES=1 \
+  RECOVER_AFTER_TIME=5m \
+  PLUGIN_LOGLEVEL=INFO
+
+LABEL io.k8s.description="Elasticsearch container for allowing indexing and searching of aggregated logs" \
+  io.k8s.display-name="Elasticsearch ${ES_VER}" \
+  io.openshift.expose-services="9200:https, 9300:https" \
```

Comment:
I don't know what OSE does.  For example, here is the ES Dockerfile used by origin: https://github.com/openshift/origin-integration-common/blob/master/elasticsearch/Dockerfile#L20 - I'm assuming OSE uses it for something . . .